concourse/pipelines/guest-package-build.jsonnet

```diff
@@ -932,31 +932,32 @@ local buildpackageimagetask = {
     },
     buildpackagejob {
       package: 'artifact-registry-yum-plugin',
-      builds: ['el7', 'el8', 'el9'],
+      builds: ['el7', 'el8', 'el8-arm64', 'el9', 'el9-arm64'],
```

Comment:
Since its noarch, we don't need to build for arm64 specifically.

Reply:
as discussed these are arch specific, so arm64 is required as well